xml/hwbp_registers.xml

```diff
@@ -1132,18 +1132,20 @@
     <register name="Interrupt Trigger" short="itrigger" address="0x7a1">
         This register is accessible as \RcsrTdataOne when \FcsrTdataOneType is 4.
 
-        This trigger may fire on any of the interrupts configurable in \Rmie
-        (described in the Privileged Spec) or the NMI. The interrupts to fire on are
-        configured by setting the same bit in \RcsrTdataTwo as would be set in
-        \Rmie to enable the interrupt.
+        This trigger may fire on any of the interrupts described in the
+        Privileged Spec, or the NMI. The interrupts to fire on are configured by
+        setting the bit in \RcsrTdataTwo whose index would be written to the
+        Exception Code field of the mode-relevant {\tt cause} register when the
+        trap is taken. For interrupts whose numbers are translated, that means
+        the number checked is the one in the mode that the trap handler executes
+        in.
```

Comment:
I don't think that this sentence will make sense to people who aren't familiar with AIA.  I would say something like:
```suggestion
        trap is taken. For virtualized interrupts that have different interrupt numbers
        when viewed from different modes, that means
        the number checked is the one in the mode that the trap handler executes
        in.
```

Reply:
Do we also need to say something about how we don't care why the interrupt is taken?  It could be that the input wire asserted, it could be that M mode wrote mie.STIE=1 in the original priv arch, it could be that the hypervisor is playing with hvip, etc.  This was one of John's concerns.  We don't have to spell out the cases but we should say something about how the trigger is based on the interrupt trap regardless of the original cause of the interrupt.

Reply:
I kind of agree with you, but I'm not sure how to say that. Aren't interrupt traps by definition taken in response to interrupts?

Reply:
I think that John's question was whether one category of reasons for taking an interrupt would be treated differently than another category of reasons for taking an interrupt.

Rereading what's currently there, it basically says that the trigger matches/fires only if we trap but it doesn't say that it fires every time that we trap.  Maybe it's as simple as saying "The trigger fires if and only if the hart takes a trap because of the interrupt."  That would presumably clarify that the reason for the trap doesn't matter.